tests/pipeline-all-amps/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+IMAGE_TAG=7.0.0-M2
+
+sed -e "s@\${image.tag}@$IMAGE_TAG@" Dockerfile > target/Dockerfile
+cp ../docker-entrypoint.sh target/
+
+docker rmi -f $DOCKER_IMAGE_REPO
+docker build -f target/Dockerfile -t $DOCKER_IMAGE_REPO .
+echo http://localhost:8080$BASE_PATH
+# --user 1000:1000 \
```

Comment:
I guess we can delete this.

Reply:
meant for local testing, ideally this should work with the unprivileged user commented but this requires the alfresco-mmt to be fixed in order not to touch anything outside the webapp

Reply:
I should create an issue for that as the code is not straightforward for something that in the end should just unpack a zip in a dir